package/src/common/configure.ts

```diff
@@ -53,19 +53,21 @@ export async function configure(
     );
   }
 
+  info("Copied Quarto script to bin directory");
+
   // record dev config
   const devConfig = createDevConfig(
-    Deno.env.get("DENO") || "",
-    Deno.env.get("DENO_DOM") || "",
-    Deno.env.get("PANDOC") || "",
-    Deno.env.get("DARTSASS") || "",
-    Deno.env.get("ESBUILD") || "",
+    Deno.env.get("QUARTO_DENO_PATH") || "",
```

Comment:
These would be set with a conda activation script if providing external executables. The activation script would probably be part of the quarto conda package, as opposed to an activation script in each of the other packages. Generally, it's better to rely on PATH, but using these here lets us manually specify the potentially vendored paths.